controllers/articles.controller.js

```diff
@@ -27,9 +27,26 @@ exports.patchArticleById = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-  selectArticles().then((articles) => {
-    res.status(200).send({ articles });
-  });
+  if (Object.keys(req.query).length > 0) {
+    if (
+      req.query.hasOwnProperty('sort_by') ||
+      req.query.hasOwnProperty('order_by') ||
+      req.query.hasOwnProperty('topic')
+    ) {
+    } else {
+      return Promise.reject({ status: 400, msg: 'Misspelt query!' }).catch(
+        next
+      );
```

Comment:
don't need to append a catch onto the promise.reject

Reply:
Deleted this catch